storage/SystemInfo/CmdBtrfs.cc

```diff
@@ -194,19 +192,59 @@ namespace storage
 
 
     std::ostream&
-    operator<<(std::ostream& s, const CmdBtrfsSubvolumes& cmdbtrfssubvolumes)
+    operator<<(std::ostream& s, const CmdBtrfsSubvolumeList& cmdbtrfssubvolumelist)
+    {
+	for (const CmdBtrfsSubvolumeList::Entry& entry : cmdbtrfssubvolumelist)
+	    s << entry;
+
+	return s;
+    }
+
+
+    std::ostream&
+    operator<<(std::ostream& s, const CmdBtrfsSubvolumeList::Entry& entry)
     {
-	for (const CmdBtrfsSubvolumes::Entry& entry : cmdbtrfssubvolumes)
-	    s << "data " << entry.path << '\n';
+	s << "id:" << entry.id << " parent-id:" << entry.parent_id
+	  << " path:" << entry.path << '\n';
 
 	return s;
     }
 
 
+    CmdBtrfsSubvolumeGetDefault::CmdBtrfsSubvolumeGetDefault(const string& mount_point)
+	: id(-1 /* BtrfsSubvolume::Impl::unknown_id */)
```

Comment:
Does `BtrfsSubvolume::Impl::unknown_id` exist? or is it only a reference for future implementation?